src/lbaf/IO/lbsVTDataWriter.py

```diff
@@ -17,35 +16,34 @@ class VTDataWriter:
         of MPI ranks that VT is utilizing.
     """
 
-    def __init__(self, phase: Phase, logger: Logger, f: str = "lbs_out", s: str = "json", output_dir=None):
+    def __init__(
+        self,
+        logger: Logger,
+        output_dir: str,
+        stem: str,
+        parameters: dict):
         """ Class constructor:
             phase: Phase instance
-            f: file name stem
-            s: suffix
+            stem: file name stem
+            parameters: a dictionary of parameters
         """
         # Assign logger to instance variable
         self.__logger = logger
 
-        # Ensure that provided phase has correct type
-        if not isinstance(phase, Phase):
-            self.__logger.error("Could not write to ExodusII file by lack of a LBS phase")
-            return
-
         # Assign internals
-        self.__phase = phase
-        self.__file_stem = f"{f}"
-        self.__suffix = s
-        self.__output_dir = output_dir
+        self.__file_stem = stem
+        if output_dir is not None:
+            self.__file_stem = os.path.join(
+                output_dir, self.__file_stem)
+        try:
+            self.__extension = parameters["json_output_suffix"]
+            self.__compress = parameters["compressed"]
+        except Exception as ex:
+            self.logger.error("Missing JSON writer configuration parameter(s): %s", ex)
+            sys.excepthook = exc_handler
+            raise SystemExit(1) from ex
 
-    def write(self):
-        """ Write one JSON file per rank."""
-        sys.setrecursionlimit(25000)
-        with Pool(context=get_context("fork")) as pool:
-            results = pool.imap_unordered(self.json_writer, self.__phase.get_ranks())
-            for file_name in results:
-                self.__logger.info(f"Saved {file_name}")
-
-    def __create_object_entries(self, rank_id, objects):
+    def __create_tasks(self, rank_id, objects):
```

Comment:
This still outputs only a subset of the potential task attributes that may have been included in the input file. If something appears in the input file, I think we should retain it even if LBAF doesn't know what it means. What is the plan for moving in that direction?

Reply:
Good question! I think we need to discuss what we want to do with it, whether we want it to be fully-fledged or not (given the fact that the vt C++ version will supersede it). Can we put this on the agenda for the upcoming LB discussion?

Reply:
Following our discussion of 05/03 I believe this is is resolved